### ISSUE TITLE
(feature request) Mouse rotation scene3d

### ISSUE THREAD

**Issue by rsmarinho** [veusz/veusz#590]
I wonder if it would be possible to use the mouse middle button to control the X, Y, Z rotation of the scene3d page component. 
It is a bit annoying to numerically change the scene rotation.

**Comment by jeremysanders** [comment]
Yes, I agree. Some kind of mouse control would be great. This is intended, but I need to work out how to do the maths to convert a mouse move into a rotation.

**Comment by rsmarinho** [comment]
At first, I thought that direct mapping XY mouse movement to XY scene rotation would work. But then I realize it is not a XY to XY simply mapping. 

I'd say It is more like mapping the XYZ camera movement, not the object...

**Comment by srk** [comment]
Seconded for request on this feature
